packages/cli/test/utils/simulation/LodestarBeaconNodeProcess.ts

```diff
@@ -67,6 +67,9 @@ export const LodestarBeaconNodeProcess: BeaconNodeConstructor = class LodestarBe
       "params.BELLATRIX_FORK_EPOCH": String(this.params.bellatrixEpoch),
       logPrefix: this.id,
       logFormatGenesisTime: `${this.params.genesisTime}`,
+      logFile: `${this.params.logFilesDir}/${this.id}.log`,
+      logFileLevel: "debug",
+      logLevel: process.env.SHOW_LOGS ? "info" : "error",
```

Comment:
This is not enough, log info to sub-process stdout and don't pipe std to main process

Reply:
If we don't pipe std of child process to main process, then it does not matter what its been logged in the child process at all.

I believe this is better approach for someone to easily debug in development.